pkg/config/crd/store.go

```diff
@@ -198,13 +237,16 @@ func toEvent(t store.ChangeType, obj interface{}) store.BackendEvent {
 }
 
 func (s *Store) dispatch(ev store.BackendEvent) {
+	s.watchMutex.RLock()
```

Comment:
just `defer s.WatchMutext.RUnlock()` and remove the other 2

Reply:
done